src/query.rs

```diff
@@ -195,10 +224,14 @@ impl<'a> ElementQuery<'a> {
         self.add_selector(ElementSelector::new(by))
     }
 
+    //
+    // Retrivers
+    //
+
     /// Return true if an element matches any selector, otherwise false.
     /// This method will not wait, and will not mutate the underlying ElementQuery.
     pub async fn exists(&self) -> WebDriverResult<bool> {
-        let elements = self.run_poller_with_options(None, None, 0).await?;
+        let elements = self.run_poller().await?;
```

Comment:
I think you want the `run_poller_with_options` here to force a non-waiting ElementPoller. The intention was for the `exists()` check to always be immediate regardless of the default poller in place. Let me know if you think it should work differently.

Reply:
The thought process I had was if someone does:
```driver.query(By:Css("#id")).wait(Duration::from_secs(), Duration::from_milis(10)).exists().await?```
It should wait for the element to maybe pop into existence before checking, but if it pops up before that then it should return emidiately. Like a fancy first() that returns a bool (May be obsolete).

The only case it wouldn't make sense is if the user defined
```driver.config_mut().set("ElementPoller", poller)?; ```
Since it wold feel like a side effect of having a configuration like that.